ansible-ipi-install/roles/node-prep/tasks/100_power_off_cluster_servers.yml

```diff
@@ -19,5 +19,9 @@
         password: "{{ hostvars[item]['ipmi_password'] }}"
         port: "{{ hostvars[item]['ipmi_port'] | default(623) }}"
         state: off
+      register: power_off_hosts
+      until: power_off_hosts is not failed
+      retries: 10
+      delay: 5
```

Comment:
do we really want that many retries?

Reply:
Yes, we do. If it fails with these parameters it really needs manual interaction resetting the idracs, otherwise up to 10 retries should do the job.